angular/.storybook/presets.js

```diff
@@ -0,0 +1,4 @@
+module.exports = [
+  '@storybook/addon-docs/angular/preset',
+  '@storybook/preset-scss'
```

Comment:
should do it manually so that postcss-loader can get used

Reply:
The angular SB doesnt allow full-control mode so we'll have to work on this after our thing.